src/typings/tacoLiveReload.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../typings/Q.d.ts" />
+
+declare module TacoLiveReload {
+
+    export function start(projectRoot: string, platforms: string[], options: LiveReloadOptions): Q.Promise<any>;
+    export function on(eventName: string, cb: (...args : any[]) => void): void;
```

Comment:
Typescript currently doesn't support support string enums.
And doing something like the below example doesn't work. Any idea ?

```
    export class Events {
        static ERROR= "livereload:error";
    }
```